src/pages/ProfileForm/index.tsx

```diff
@@ -24,19 +32,92 @@ function getStepForm(step = 1): () => JSX.Element {
 }
 
 /**
- * 
+ *
  * @returns Chooses which StepForm to return based on the store value
  */
-function StepForm(): JSX.Element {
-  const step = useProfileForm((state) => state.step);
+function StepForm(): JSX.Element | null {
+  const ProfileFormState = useProfileForm;
+  const redirect500 = useError500();
+
+  const step = useProfileForm(state => state.step);
+  const auth = useSblAuth();
+  const { emailDomain, isLoading: authIsLoading } = auth;
+
```

Comment:
Heyo @shindigira, not a biggie at all, but it looks like this is actually a bug not in this PR but with [your earlier Axios PR](https://github.com/cfpb/sbl-frontend/pull/207/files#diff-10588f64be799cd6c3a4b42df38020c26a12b2bf48afe582174d0a1ef502eb8fL7-L9) where the wrong URL was maybe being used for `isDomainAllowed?` It's just happening now because I merged with `main`, and sorry for not catching this during code review.



Here's a fix:
https://github.com/cfpb/sbl-frontend/pull/231